### ISSUE TITLE
Getting image error

### ISSUE THREAD

**Issue by vaibhavkansara** [terraform-google-modules/cloud-foundation-training#55]
### TL;DR

While using file from the path (https://github.com/terraform-google-modules/cloud-foundation-training/tree/master/Solutions)/04-Instance-Group/main.tf.solution file .

Getting below error 

│ Error: Error resolving image name 'debian-cloud/debian-9': Could not find image or family debian-cloud/debian-9
│
│   with module.instance_template.google_compute_instance_template.tpl,
│   on .terraform/modules/instance_template/modules/instance_template/main.tf line 58, in resource "google_compute_instance_template" "tpl":
│   58: resource "google_compute_instance_template" "tpl" {

### Expected behavior

Should work fine

### Observed behavior

Getting below error

│ Error: Error resolving image name 'debian-cloud/debian-9': Could not find image or family debian-cloud/debian-9
│
│   with module.instance_template.google_compute_instance_template.tpl,
│   on .terraform/modules/instance_template/modules/instance_template/main.tf line 58, in resource "google_compute_instance_template" "tpl":
│   58: resource "google_compute_instance_template" "tpl" {

### Terraform Configuration

```hcl
(qwiklabs-gcp-03-9a00e2ed0b4f)$ terraform init
Initializing modules...

Initializing the backend...

Initializing provider plugins...
- Reusing previous version of hashicorp/google-beta from the dependency lock file
- Reusing previous version of hashicorp/google from the dependency lock file
- Reusing previous version of hashicorp/local from the dependency lock file
- Reusing previous version of hashicorp/random from the dependency lock file
- Using previously-installed hashicorp/google-beta v4.31.0
- Using previously-installed hashicorp/google v4.31.0
- Using previously-installed hashicorp/local v2.2.3
- Using previously-installed hashicorp/random v3.3.2

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
```

### Terraform Version

```sh
(qwiklabs-gcp-03-9a00e2ed0b4f)$ terraform init
Initializing modules...

Initializing the backend...

Initializing provider plugins...
- Reusing previous version of hashicorp/google-beta from the dependency lock file
- Reusing previous version of hashicorp/google from the dependency lock file
- Reusing previous version of hashicorp/local from the dependency lock file
- Reusing previous version of hashicorp/random from the dependency lock file
- Using previously-installed hashicorp/google-beta v4.31.0
- Using previously-installed hashicorp/google v4.31.0
- Using previously-installed hashicorp/local v2.2.3
- Using previously-installed hashicorp/random v3.3.2

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
```

### Additional information

from the file(main.tf.soution) after removing the  line 75 and 76 (  source_image_family  = "debian-9"
  source_image_project = "debian-cloud").Re-initiate the terraform and re-applied the plan and  it works fine.

**Comment by dalviparth** [comment]
Hi @vaibhavkansara, I have rectify the error you are facing in this lab. You just have to change  **source_image_family  = "debian-9"** to  **source_image_family  = "debian-10"**

module "instance_template" {
  source               = "terraform-google-modules/vm/google//modules/instance_template"
  version              = "~> 7.5.0"
  project_id           = module.project_iam_bindings.projects[0]
  subnetwork           = module.network.subnets_self_links[0]
  source_image_family  = "debian-10"
  source_image_project = "debian-cloud"
  startup_script       = data.local_file.instance_startup_script.content
  service_account = {
    email  = google_service_account.instance_group.email
    scopes = ["cloud-platform"]
  }
  tags = ["allow-load-balancer"]
}

**Comment by Rajarshi-Ghosh** [comment]
Yes, seems like debian 9 is not available for some reason, use debian 10 instead. Thanks!
